wallet/core/src/utxo/processor.rs

```diff
@@ -604,9 +604,7 @@ impl UtxoProcessor {
             match kind {
                 MetricsUpdateKind::WalletMetrics => {
                     let mempool_size = snapshot.get(&Metric::NetworkMempoolSize) as u64;
-                    let node_peers = snapshot.get(&Metric::NodeActivePeers) as u32;
-                    let network_tps = snapshot.get(&Metric::NetworkTransactionsPerSecond);
-                    let metrics = MetricsUpdate::WalletMetrics { mempool_size, node_peers, network_tps };
+                    let metrics = MetricsUpdate::WalletMetrics { mempool_size };
```

Comment:
Was the removal of the node_peers and network_tps here intentional? And why?

Reply:
cleanup. this code is not currently active and the APIs that use similar code in KNG, only utilize mempool_size.

This change is a victim of documentation pass (couldn't document what shouldn't be there).